helm/charts/kratos/templates/secrets.yaml

```
@@ -9,4 +9,4 @@ type: Opaque
 data:
   dsn: {{ include "kratos.dsn" . | b64enc | quote }}
   # Generate a random secret if the user doesn't give one. User given password has priority
-  secretsSession: {{ include "kratos.secrets.session" . | required "Value kratos.config.secrets.session can not be empty!" | b64enc | quote }}
+  secretsSession: {{ include "kratos.secrets.session" . | b64enc | quote }}
```

Even though the secret is not required in Kratos, it's a bad idea for this helm chart to not require it. Without a fixed secret, you will not be able to scale Kratos nor will you be able to restart it, because your users will all be logged out!

Therefore, I suggest we revert this change:

```suggestion
secretsSession: {{ include "kratos.secrets.session" . | required "Value kratos.config.secrets.session can not be empty!" | b64enc | quote }}
```